doc/nrf/releases/release-notes-latest.rst

```diff
@@ -20,8 +20,18 @@ See the change log for each library in the :doc:`nrfxlib documentation <nrfxlib:
 sdk-zephyr
 ==========
 
+.. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.
+
 The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``4ef29b34e3``, plus some |NCS| specific additions.
 
+For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run::
+
+   git log --oneline 4ef29b34e3 ^v2.3.0-rc1-ncs1
```

Comment:
Won't this include the NCS-specific commits with sauce tags? I was thinking something along the lines of `git log <last_release_upstream_sha>..<this_release_upstream_sha>`

Reply:
Scratch that, I confused myself now, this is correct.